uk.co.compucorp.civicrm.hrcore/backstop_data/scenarios/ssp-la-my-leave-calendar.json

```diff
@@ -0,0 +1,14 @@
+{
+  "scenarios": [
+    {
+      "label": "Current Month: My Leave Calendar",
+      "url": "my-leave#/my-leave/calendar",
+      "delay": 10000
```

Comment:
This is a lot of delay. Please add a script that waits for a particular element to be visible